src/main/java/uk/ac/cam/cl/dtg/segue/api/monitors/PerformanceMonitor.java

```diff
@@ -90,8 +89,11 @@ public void filter(final ContainerRequestContext requestContext, final Container
 
         // Record for metrics
         REQUEST_LATENCY_HISTOGRAM
-                .labels(requestContext.getMethod(), monitorService.getPathWithoutPathParamValues(request.getUri()))
-                .observe((double)timeInMs / NUMBER_OF_MILLISECONDS_IN_A_SECOND);
+                .labels(
+                        requestContext.getMethod(),
+                        monitorService.getPathWithoutPathParamValues(request.getUri()),
+                        String.valueOf(responseContext.getStatus())
```

Comment:
Just a note that getStatus returns an int and therefore -1 when the status is not, if that was ever the case. This is probably a good thing here because String.valueOf(...) throws a null pointer exception if it is given null!